Tools/Scripts/webkitpy/common/system/platforminfo.py

```diff
@@ -214,7 +214,11 @@ def available_sdks(self):
             return []
 
         XCODE_SDK_REGEX = re.compile(r'\-sdk (?P<sdk>\D+)\d+\.\d+(?P<specifier>\D*)')
-        output = self._executive.run_command(['xcodebuild', '-showsdks'], return_stderr=False)
+
+        try:
+            output = self._executive.run_command(['xcodebuild', '-showsdks'], return_stderr=False)
+        except:
+            output = ''
```

Comment:
I think that it would be more elegant to just return an empty list here, not to fool subsequent code.

Reply:
Done.